### ISSUE TITLE
Dismantle objects with multiple relationships in the graphic view

### ISSUE THREAD

**Issue by nnhjy** [Spine-project/Spine-Toolbox#1743]
As the graph below presents, the object `ELC` causes annoying crossings over other relationships. My initial thought on this is to enable dismantling the `ELC` object (only) in the graph per relationship it has. The dismantled `ELC` graphic objects refer to the same `ELC` defined in the DB. 

How difficult to implement this magic?
![1660839378286](https://user-images.githubusercontent.com/43530784/185444262-0fa69b80-e668-4c80-9609-2306b450ff08.png)

**Comment by soininen** [comment]
There is the option "Hide selected" that is accessible from the right click context menu. Does it do work in this case?

**Comment by nnhjy** [comment]
Yeah, it works to some extent. By hiding the `ELC` node, only the relationship remains. This simplifies the graph but hides a bit of the information that all those inputs relationships without a node, which look different, actually represent the same input `ELC`. Would there be a way by which we can keep some key info of hidden items?
